packages/spectacular/src/lib/feature-testing/configuration/spectacular-feature-testing-features.ts

```diff
@@ -0,0 +1,53 @@
+import type { Provider } from '@angular/core';
+import type { provideSpectacularFeatureTesting } from './provide-spectacular-feature-testing';
```

Comment:
nitpick: nice warning from the linter that `provideSpectacularFeatureTesting` is not being used

Reply:
It's used by TSDoc `@link` annotations.